src/fluree/server/handler.clj

```diff
@@ -165,26 +171,37 @@
   as :body-params. If the signature is not valid, throws an invalid signature error. If
   the request body is not a credential, nothing is done."
   [handler]
-  (fn [{:keys [body-params] :as req}]
+  (fn [{:keys [body-params content-type] :as req}]
     (log/trace "unwrap-credential body-params:" body-params)
-    (let [verified (<!! (cred/verify body-params))
-          _        (log/trace "unwrap-credential verified:" verified)
-          {:keys [subject did]}
-          (cond
-            (:subject verified) ; valid credential
-            verified
-
-            (and (util/exception? verified)
-                 (not= 400 (-> verified ex-data :status))) ; no credential
-            {:subject body-params}
-
-            :else ; invalid credential
-            (throw (ex-info "Invalid credential"
-                            {:response {:status 400
-                                        :body   {:error "Invalid credential"}}})))
-          req*     (assoc req :body-params subject :credential/did did :raw-txn body-params)]
-      (log/debug "Unwrapped credential with did:" did)
-      (handler req*))))
+    (if (#{"application/json" "application/jwt"} content-type)
```

Comment:
Is it possible to sign a turtle import? I think you'd use a JWT, but `cred/verify` will not handle turtle correctly.

Reply:
I think so, but it wasn't the priority and I think there is a good chance this middleware dissapears, but not 100% on that.

The db api now has `*-credential` APIs which ideally we'd mostly leverage instead of fluree/server doing its own thing.

But for now it doesn't have signatures for TTL imports, although I should make a ticket for it as I think it should at some point.